src/char/inter.c

```diff
@@ -814,19 +814,27 @@ static int inter_config_read(const char* cfgName)
 	return 0;
 }
 
-// Save interlog into sql
+/**
+* Save interlog values to SQL
+* @param fmt: Pointer containing the type of activity to be stored
+*  Supplied arguments: activity, origin, target, guild pointer, castle_id
+* @return 0
+*/
 int inter_log(char* fmt, ...)
```

Comment:
Because a lot of code in *athena wrote by people who wish just show their skills in coding and show how they are good, without thinking that others must read their codes and continue development.